sdk/storage/README.md

```diff
@@ -20,6 +20,32 @@ The Azure Storage Client Library for C++ allows you to build applications agains
 
 For the best development experience, we recommend that developers use the [CMake projects in Visual Studio](https://docs.microsoft.com/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) to view and build the source code together with its dependencies.
 
+### Include the package
```

Comment:
how about moving this section to https://github.com/Azure/azure-sdk-for-cpp/blob/e9cebfa105440beec7f9675c201854b7d4a77c2b/sdk/storage/README.md#via-vcpkg